### ISSUE TITLE
The template .tex file still refers to R and sweave

### ISSUE THREAD

**Issue by shankari** [chiehrosswang/TRB_LaTeX_tex#5]
According to the README,
> The previous version was built primarily for people using R, Sweave, and LaTeX. Therefore, this version is created for people who need a straight-forward LaTeX template for TRB papers.

And the details on how to use it are at
> More instruction and details about the project can be found inside the trb_template.tex.

But the template still has R and sweave instructions.
https://github.com/chiehrosswang/TRB_LaTeX_tex/blob/master/trb_template.tex#L196

The instructions in the README

```
latexmk trb_template.tex -pdf -pvc -shell-escape
```
seems to work fine.

I'll send a pull request once I've verified that the other instructions work.

**Comment by chiehrosswang** [comment]
You're right, will need to update the content of the document.  But like you said, `latexmk trb_template.tex -pdf -pvc -shell-escape` should still work.  Will wait for your pull request.

**Comment by chiehrosswang** [comment]
This is has been addressed in recent updates.
